src/test/java/com/powsybl/openloadflow/sa/OpenSecurityAnalysisTest.java

```diff
@@ -387,13 +384,13 @@ void testHighVoltageLimitViolations() {
     void testActivePowerLimitViolations() {
         Network network = createNodeBreakerNetwork();
         network.getLine("L1").newActivePowerLimits1()
-               .setPermanentLimit(1.0)
-               .beginTemporaryLimit()
-               .setName("60")
-               .setAcceptableDuration(60)
-               .setValue(1.2)
-               .endTemporaryLimit()
-               .add();
+                .setPermanentLimit(1.0)
```

Comment:
@geofjamg I have an issue with indentation... and I don't know how to revert this. I can dot it by hand if no other solution...